### DISCUSSION TITLE
How should joined conferences be defined?

### DISCUSSION THREAD

**Discussion by tholzheim** [SmartDataAnalytics/OpenResearch#143] [Q&A]
Currently the form and template of event only support one event for the [property series](https://www.openresearch.org/wiki/Property:Event_in_series).
Joint conferences are therefore only assigned to one series.
![image](https://user-images.githubusercontent.com/45004115/110106072-8046ca00-7da9-11eb-8189-b731ce5ace32.png)
To create the reference to the other series the information is added to the free text.
```
This conference belongs additionally to the series [[Event in series::PRICAI]]
```
Resulting in:
![image](https://user-images.githubusercontent.com/45004115/110106271-c4d26580-7da9-11eb-9f6a-8bd390fe7cb4.png)
Meaning that two series are assigned to a joint conference but only one of them is supported by the form and template.

What is the desired definition of joint conferences?
* two series, or
* separate property

**Comment by WolfgangFahl** [comment]
The standard case is that an event belongs to exactly one series. To keep the complexity of the system low i suggest that any non standard case is marked as such. Such an event will be linked to one of the possible series and marked as a special case. This scenario even led to the discusson in wikidata: https://www.wikidata.org/wiki/Wikidata:Property_proposal/unusualness where https://www.wikidata.org/wiki/Q7395156 shows how horrible an implementation can get if non-standard cases are implemented in an awkward way. I'd rather keep this out of the next milestone and concentrate on the standard case.
